armory/data/adversarial/ucf101_mars_perturbation_and_patch_adversarial_112x112.py

```diff
@@ -171,11 +171,17 @@ def _info(self):
                             shape=(None, 112, 112, 3)
                         ),
                     },
-                    "label": tfds.features.ClassLabel(names=_LABELS),
+                    "labels": {
+                        "clean": tfds.features.ClassLabel(names=_LABELS),
+                        "adversarial_perturbation": tfds.features.ClassLabel(
+                            names=_LABELS
+                        ),
+                        "adversarial_patch": tfds.features.ClassLabel(names=_LABELS),
+                    },
```

Comment:
This looks like it will break existing things that currently use the adversarial dataset, yes?

So merging this is blocked on updates to the how these loaders work.

Reply:
Do we have similar targeted labels for the other adversarial datasets?

Reply:
This, this will break existing things.  This is an example of how we could pair the labels, but not necessarily the final version.  To separate out the labels, I imagine you could use a method similar to that used to separate out the clean/adv inputs?

Image and audio will also have targeted labels, but the audio targeted adversarial dataset is not ready, so the current delivery only includes clean and untargeted examples.  Going forward, for most scenarios will have both targeted and untargeted examples

Reply:
We can discuss more tomorrow on how you'd want to proceed with this